src/tensor_ops.jl

```diff
@@ -256,6 +256,23 @@ end
 Base.ctranspose(S::AllTensors) = transpose(S)
 
 
+############################
+# Symmetric/Skew-symmetric #
+############################
+
+"""
+Computes the symmetric part of a second order tensor.
+"""
+@inline sym{dim,T}(S1::Tensor{2,dim,T}) = convert(SymmetricTensor{2,dim,T},S1)
```

Comment:
Not sure if this should be a `Tensor` or not. Operators that swap the type willy nilly can be a bit annoying and you can easily just do the conversion yourself if you want a SymmetricTensor type.